internal/util/printerutil/printerutil.go

```diff
@@ -21,12 +21,9 @@ import (
 )
 
 // PrintFnResultInfo displays information about the function results file.
-func PrintFnResultInfo(ctx context.Context, resultsFile string, withNewLine, toStdErr bool) {
+func PrintFnResultInfo(ctx context.Context, resultsFile string, withNewLine bool) {
 	pr := printer.FromContextOrDie(ctx)
-	var opt *printer.Options
-	if toStdErr {
-		opt = printer.NewOpt().Stderr()
-	}
+	opt := printer.NewOpt().Stderr()
```

Comment:
If printer supports only writing to one stream, this option becomes redundant.

Reply:
There is no option there. It is only writing to stderr stream.

Reply:
Updated.